index.html

```diff
@@ -172,7 +172,7 @@
           <dd>This method returns a <a>PerformanceEntryList</a> object that contains a copy of <a>PerformanceEntry</a> objects, in chronological order with respect to <a href="#widl-PerformanceEntry-startTime">startTime</a>, that have the same value for the <a href="#widl-PerformanceEntry-name">name</a> attribute of <a>PerformanceEntry</a> as the <a href="#widl-PerformanceEntry-name">name</a> parameter and, if specified, have the same value for the <a href="#widl-PerformanceEntry-entryType">entryType</a> attribute of <a>PerformanceEntry</a> as the <a href="#widl-PerformanceEntry-entryType">entryType</a> parameter. If no such <a>PerformanceEntry</a> objects exist, the <a>PerformanceEntryList</a> must be empty.</dd>
         </dl>
 
-        <dl class="idl" title="typedef sequence PerformanceEntry PerformanceEntryList"></dl>
+        <dl class="idl" title="typedef sequence&lt;PerformanceEntry&gt; PerformanceEntryList"></dl>
```

Comment:
FWIW, I don't think you need to escape this, respec should do the right thing: https://www.w3.org/respec/examples/webidl.html#typedefs